src/test/java/org/broadinstitute/hellbender/tools/exome/ExomeToolsTestUtils.java

```diff
@@ -34,42 +34,57 @@ protected static File getTestDataDir(){
      */
     protected final static SAMSequenceDictionary REFERENCE_DICTIONARY = SAMSequenceDictionaryExtractor.extractDictionary(REFERENCE_FILE);
 
+    protected static Comparator<SimpleInterval> createIntervalComparator() {
```

Comment:
We didn't make it comparable because there isn't an inherent ordering on them.  I think it would be fine to provide a comparator though.